docs/integration.rst

```diff
@@ -314,6 +314,12 @@ selected) you can also give the name of the slot::
 
   rauc.slot=recovery.0
 
+When booting from a source not configured your system.conf (for example from a
+USB memory stick), you can tell rauc explicitly with the flag ``
+rauc.external``.
+This means that all slots are known to be inactive and will be valid
+installation targets.
```

Comment:
Vielleicht noch ein Hinweise à la:

"A common use case for this is to use RAUC during bootstrapping procedure to initially fill up slots."